src/components/SpaceProposalNFTMintModalProgress.vue

```diff
@@ -0,0 +1,86 @@
+<script setup lang="ts">
+const { progress, Status } = useNFTClaimerProgress();
+
+const length = computed(() => {
+  return Object.keys(progress).length;
+});
+
+function nextStepStatus(step: number) {
+  const steps = Object.keys(progress.value);
+  if (step + 1 > steps.length - 1) {
+    return false;
+  }
+
+  return progress.value[steps[step + 1]].status;
+}
+</script>
+
+<template>
+  <nav class="flex" aria-label="Progress">
+    <ol role="list" class="space-y-4 pt-3">
+      <li v-for="(step, key, i) in progress" :key="key">
```

Comment:
The last to items don't have a connecting line

Reply:
<img width="389" alt="image" src="https://github.com/snapshot-labs/snapshot/assets/51686767/21eb8c35-2a14-4f95-b67a-4cfa893b16bf">
<img width="610" alt="Screenshot 2023-07-18 at 7 03 12 AM" src="https://github.com/snapshot-labs/snapshot/assets/51686767/cf3fd1eb-b47d-4b9a-a943-d780b568d792">